tests/vm_test.go

```diff
@@ -122,6 +122,16 @@ var _ = Describe("[rfe_id:1177][crit:medium][vendor:cnv-qe@redhat.com][level:com
 			Expect(reviewResponse.Details.Causes).To(HaveLen(1))
 			Expect(reviewResponse.Details.Causes[0].Field).To(Equal("spec.template.spec.domain.devices.disks[2].name"))
 		})
+		It("should reject creation if template spec does not contain any memory settings", func() {
```

Comment:
Actually, the comment ` It is only meant to be used for VMI reviews, not if they are templates on other objects`  still holds and we should not merge this. It is because there are still presets around...

Reply:
The only use of `ValidateVirtualMachineInstanceMandatoryFields` I could find was in the `Admit` method of the `vmi-create-admitter`. Therefore I concluded it would be ok to remove this comment and use it also in the `vm-admitter`. Can you show me where this is used in presets?

Reply:
So you say the vm-admitter needs to lookup VirtualMachineInstancePresets and admit the VM if there are no memory related settings but there was a matching preset?

Reply:
This is my expectation, I was expecting we would see a failed test as well. I did not look at the test tho... Would you mind having a look to see if we are safe from presets?

Reply:
Could we also say that instanceTypes conflcit with presets?
Mainly because presets are deprecated?